packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -301,6 +301,7 @@ function TokenListsPanel() {
                   type="checkbox"
                   checked={isActive}
                   onChange={() => toggleTokenList(tokenList, isActive)}
+                  data-cy={`toggle ${tokenList.name}`}
```

Comment:
Can we query this element any other way? IMO, querying by `data-cy` is a code smell.

https://testing-library.com/docs/queries/about/#priority

Reply:
I've gone through the priority list you linked and I can't see any of the options that could apply in this case

Reply:
Might need to use `data-cy` unfortunately unless we use something that doesn't really make sense, like `alt` text